src/gordon_gcp/plugins/enricher.py

```diff
@@ -79,6 +79,10 @@ def _validate_config(self):
             msg = 'A dns zone must be a FQDN and end with the root zone (".").'
             errors.append(msg)
 
+        if not self.config.get('default_ttl'):
+            msg = 'A default record TTL is required.'
+            errors.append(msg)
+
```

Comment:
I'm not sure this should be in the Publisher PR

Reply:
Yeah, long-term it should be in gordon core.  For now, though...

Reply:
Also, I just landed a change that will create a merge conflict here.  You'll want to rebase and fix.

Reply:
I meant in terms of being in enricher, when this PR relates to publisher.